content/en/docs/setup/best-practices/cluster-large.md

```diff
@@ -2,126 +2,109 @@
 reviewers:
 - davidopp
 - lavalamp
-title: Building large clusters
+title: Considerations for large clusters
 weight: 20
 ---
 
-## Support
-
-At {{< param "version" >}}, Kubernetes supports clusters with up to 5000 nodes. More specifically, we support configurations that meet *all* of the following criteria:
+A cluster is a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (physical
+or virtual machines) running Kubernetes agents, managed by the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+Kubernetes {{< param "version" >}} supports clusters with up to 5000 nodes. More specifically,
+Kubernetes is designed to accomodate configurations that meet *all* of the following criteria:
 
+* No more than 100 pods per node
 * No more than 5000 nodes
-* No more than 150000 total pods
+* No more than 150000 total Pods
 * No more than 300000 total containers
-* No more than 100 pods per node
-
-
-## Setup
+* No more than 100 Pods per node
 
-A cluster is a set of nodes (physical or virtual machines) running Kubernetes agents, managed by a "master" (the cluster-level control plane).
+You can scale your cluster by adding or removing nodes. The way you do this depends
+on how your cluster is deployed.
 
-Normally the number of nodes in a cluster is controlled by the value `NUM_NODES` in the platform-specific `config-default.sh` file (for example, see [GCE's `config-default.sh`](https://releases.k8s.io/{{< param "githubbranch" >}}/cluster/gce/config-default.sh)).
+When setting up a large Kubernetes cluster, consider the following issues:
```

Comment:
I think I'd like more content after the sentence "You can scale your cluster ..."
I would consider removing line 24. It seems odd to introduce the next section heading with "... the following issues:".